queries.sql

```diff
@@ -1,3 +1,17 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+
+SELECT name FROM animals WHERE date_of_birth BETWEEN '201-01-01' AND '2019-12-31';
```

Comment:
- [ ] Here, you entered the wrong date of birth. Instead of `2016-01-01`, you entered `201-01-01`. Please fix that 👍 . The year 201 was a long time ago before pokemon existed😆 .